src/connectors/webauthn.ts

```diff
@@ -1,20 +1,22 @@
-import { getQsParam } from './common';
-import { b64Decode, buildDataString, parseWebauthnJson } from './common-webauthn';
+import { b64Decode, getQsParam } from './common';
+import { buildDataString, parseWebauthnJson } from './common-webauthn';
 
 // tslint:disable-next-line
 require('./webauthn.scss');
 
 document.addEventListener('DOMContentLoaded', () => {
     init();
 
-    const text = getQsParam('btnText');
```

Comment:
Just a heads up. If we do change the protocol, we'll need to include support for both variants for a while. To avoid old clients from getting locked out.